vendor/streams-text-encoding/text-decoder-stream.js

```diff
@@ -0,0 +1,95 @@
+// Adapted from  https://github.com/stardazed/sd-streams
```

Comment:
What's adapted? Could we add an entry to the README to document how to update this (if needed).

Reply:
Good callout. Added details with ec3049ce5709f70049d47069c4b7441462c98ac0.